tasks/packager.js

```diff
@@ -165,6 +165,18 @@ module.exports = function(grunt) {
 			// load each component into the buffer list
 			(only ? toArray(only) : set).forEach(loadComponent);
 
+			// remove unwanted dependencies
+			if (options.exclude){
+				var toExclude = toArray(options.exclude); 
+				buffer = buffer.filter(function(def){
+					var shouldKeep = toExclude.filter(function(dependency){
+						var match = dependency.match(/([^\*]+)/);
```

Comment:
not necessary, but I feel like this matcher should be a method reused elsewhere.